.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: ✨ Feature request
+description: Suggest an idea for this project
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Feature Request issues are for **feature requests** only. For debugging help or technical support, please use:
+        - The #ansible-awx channel on irc.libera.chat
```

Comment:
Same update as .github/ISSUE_TEMPLATE/bug_report.yml

Reply:
Another option might be to move these message to the "top level" inside of a config.yml file so the IRC/Google group is right at the top of the new issue screen.

Reply:
We changed this wording and added md style links.